integrations/github.js

```diff
@@ -35,14 +35,16 @@ module.exports = async function(robot, kredits) {
     });
   }
 
-  function createContribution(githubUser, amount, description, url, details) {
+  function createContribution(githubUser, date, time, amount, description, url, details) {
```

Comment:
with that many arguments we could use a object, this would allow use to use the same API in the different integrations. (the wiki integration does not use a time)